test/quantization/pt2e/test_quantize_pt2e.py

```diff
@@ -1810,6 +1810,10 @@ def forward(self, input_tensor, hidden_tensor):
             model_graph = convert_pt2e(model_graph)
             self.assertEqual(model_fx(*example_inputs), model_graph(*example_inputs))
 
+    def test_maxpool2d_int8(self):
+        x = torch.randint(1, 100, (1, 3, 3, 3), dtype=torch.int8)
```

Comment:
Is `torch.uint8` a possible input too?

Reply:
yes, this is just a sanity check, I'm not sure where should I add test yet